document/src/test/java/com/yahoo/document/update/TensorAddUpdateTest.java

```diff
@@ -10,20 +10,36 @@
 public class TensorAddUpdateTest {
 
     @Test
-    public void apply_add_update_operations() {
-        assertApplyTo("{{x:0,y:0}:1, {x:0,y:1}:2}", "{{x:0,y:2}:3}", "{{x:0,y:0}:1,{x:0,y:1}:2,{x:0,y:2}:3}");
-        assertApplyTo("{{x:0,y:0}:1, {x:0,y:1}:2}", "{{x:0,y:1}:3}", "{{x:0,y:0}:1,{x:0,y:1}:3}");
-        assertApplyTo("{{x:0,y:0}:1, {x:0,y:1}:2}", "{{x:0,y:1}:3,{x:0,y:2}:4}", "{{x:0,y:0}:1,{x:0,y:1}:3,{x:0,y:2}:4}");
-        assertApplyTo("{}", "{{x:0,y:0}:5}", "{{x:0,y:0}:5}");
-        assertApplyTo("{{x:0,y:0}:1, {x:0,y:1}:2}", "{}", "{{x:0,y:0}:1, {x:0,y:1}:2}");
+    public void apply_add_update_operations_sparse() {
```

Comment:
Now that merge() is implemented in the tensor library I think we should move all the detailed tests there. Only the simple integration in applyTo() should be tested here.